pkg/apis/batch/validation/validation.go

```diff
@@ -31,6 +31,8 @@ import (
 	apivalidation "k8s.io/kubernetes/pkg/apis/core/validation"
 )
 
+const maxParallelismForIndexedJob = 100000
```

Comment:
add a comment explaining why parallelism is now capped.

Reply:
Done